test/test_transforms_v2_refactored.py

```diff
@@ -396,6 +396,8 @@ def check_transform(transform, input, check_v1_compatibility=True, check_sample_
     if check_v1_compatibility:
         _check_transform_v1_compatibility(transform, input, **_to_tolerances(check_v1_compatibility))
 
+    return output
```

Comment:
We already compute the `output` in `check_transform`. By returning it, we don't need to recompute in case the test performs additional checks.